packages/twitch-chat-client/src/ChatClient.ts

```diff
@@ -501,7 +501,8 @@ export default class ChatClient extends IRCClient {
 				// unhost
 				this.emit(this.onUnhost, channel);
 			} else {
-				this.emit(this.onHost, channel, target, viewers ? Number(viewers) : undefined);
+				const numViewers = Number(viewers);
```

Comment:
In either the case that `viewers` is undefined or the case that `viewers` is a hyphen `-`, `numViewers` will be `NaN`.